hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
```

Comment:
> You're saying it works for php files but not for js files?

nope I just thought maybe you front end guys use a different config file that use 4 indentation for s files.